ProblemSolvingMentoringSession/Part 1/IntegersAcceptorChallenge/Solvings/IntegersAcceptorChallenge_Bilyk.swift

```diff
@@ -0,0 +1,14 @@
+//  Created by Maxos on 5/10/23.
+
+import Foundation
+
+final class IntegersAcceptorChallenge_Bilyk: IntegersAcceptorChallengeProtocol {
+    
+    func solve(value1: Int, value2: Int) -> Bool {
+        if value1 == 20 || value2 == 20 || value1 + value2 == 20 {
```

Comment:
Code inside if is conditional  oolean logic. In simple cases like this, you can return just this condition without if else construction